lib/utils/environment.ts

```diff
@@ -74,7 +75,8 @@ export function BuildCommand(command: string, args?: string[], options?: Command
  * @throws        An error containing a message from the failed command's `stderr` output.
  */
 export function ExecuteCommand(command: Command): Promise<{}> {
-	let tries = ((command.options || {}).retries || 0) + 1;
+	let tries = (command.options.retries || 0) + 1;
+	let delay = command.options.delay || 1000;
```

Comment:
Not sure if I mentioned this in the last comments, but you can use `_.get` here.